### DISCUSSION TITLE
What is taking so long to get MAUI on windows ARM

### DISCUSSION THREAD

**Discussion by cconner100** [dotnet/maui#13341] [General] [answered]
Seems like this is taking forever to get released.  I would like to move to my new machine for all development but am waiting on MAUI on windows ARM.  

Any idea on how long this will be

**Comment by davidortinau** [comment]
I'm running a preview of it now. It'll be in your hands very soon with VS Preview.

**Reply by JohnHDev** [reply]
@davidortinau is that release still targeting 17.6 P1? or will we get it earlier? A lot of people waiting for this one.

**Reply by Xyncgas** [reply]
What's a good arm laptop for software development?

**Reply by akhanalcs** [reply]
> What's a good arm laptop for software development?

MacBook Pro M1/ M2 Pro with at least 16 GBs of RAM

**Comment by curtis91a** [comment, marked as answer]
Its out, it works and its cool thanks guys!
